docs/AR.md

```diff
@@ -0,0 +1,131 @@
+- After entering all the information, press the save button
+
+![](./assets/ModelUpload/ModelUpload8.png)
+
+**How to upload 3D Model with Video**
+
+[![](./assets/ModelUpload/thumb.png)](https://www.youtube.com/watch?v=0i_jmbDfhes)
+
+
+## Materials that can be added to a scene
+
+- **VRX:** To upload a 3D model, you must make sure that the model is in VRX format. (Required) [Viro3DObject](https://viro-community.readme.io/docs/viro3dobject)
+- **TARGET:** If you want your 3D model to have an image recognition function, upload the desired image in this section (Optional) [ViroARImageMarker](https://viro-community.readme.io/docs/viroarimagemarker)
+- **MP3:** If you want a sound file in Augmented Reality scene, upload it to this section (Optional) [ViroSound](https://viro-community.readme.io/docs/virosound)
+- **MP4:** If you want a video on Augmented Reality scene, please upload it to this section (Optional) [ViroVideo](https://viro-community.readme.io/docs/virovideo)
+- **IMAGE:** If you want a picture in the Augmented Reality scene, upload it to this section (Optional) [ViroImage](https://viro-community.readme.io/docs/viroimage)
+- **TEXTURE:** The section where you can upload the texture files required for the 3D model (Required)
+  - **Important!** Each texture file has a specific name. Please enter this specific name in the name field. Otherwise there will be a problem in displaying the 3D model.
+
+## Customisation of the Augmented Reality scene
+
+**Note!** To view the customisation in Augmented Reality scene, you need to delete the 3D model you downloaded in the application and download it again!
+
+- **Name:** Write the name you want your 3D model to be displayed in this section (Required)
+- **Description:** Write the description you want your 3D model to be displayed in this section (Optional)
+- **Geo-Koordinaten:**
+  - **Latitude & Longitude:** Write the coordinates of your 3D model that you want to be displayed on the map in this section (Optional)
+- **Animation name:** The field where the name of the animation of the 3D model can be written (Optional)
+- **Package size:** This is the field where the total size of all Augmented Reality scene objects you have loaded should be written in bytes. The value added to this section allows the user to show how many MB files will be downloaded in the application. (Required)
+- **VRX & MP4 & IMAGE:**
+  - Set the Position, Size and Rotation properties according to x,y,z coordinates. [Scenes](https://viro-community.readme.io/docs/scenes)
+- **TARGET:**
+  - _Physical width_ - Allows you to set the distance in the physical world (in metres) between the image and the device's camera.
+- **MP3:**
+  - _Is spatial sound_ - A field that allows you to specify whether the audio file you are uploading has spatial audio capability. takes true or false value [ViroSpatialSound](https://viro-community.readme.io/docs/virospatialsound#onfinish)
```

Comment:
Is spatial sound takes only true or false values. These values currently need to be written as a string. If a checkbox is added with a future update, this can be updated.